migrations/1671299400000-Deposit.ts

```diff
@@ -0,0 +1,16 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class Deposit1671299400000 implements MigrationInterface {
+  name = "Deposit1671299400000";
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`
+      create index "IX_deposit_srAddress_depositDate_pId_tId_status" 
+        on deposit ("stickyReferralAddress", "depositDate", "priceId", "tokenId", status)
+    `);
```

Comment:
This index will help to get the deposits faster when using these fields in the WHERE clause